projects/remote_transceiver/src/remote_transceiver.cpp

```diff
@@ -85,6 +85,7 @@ void Listener::run()
 void HTTPServer::readReq()
 {
     std::shared_ptr<HTTPServer> self = shared_from_this();
+    req_.clear();
```

Comment:
I can't find any documentation for the clear() function. Where did you find it? Everything I've seen just sets `req_ = {}` to clear it: [official example](https://github.com/boostorg/beast/blob/f8ff44d072803e8c193946818ee7d239c5834b10/example/http/server/async/http_server_async.cpp#L249).